geppetto-showcase/package.json

```diff
@@ -129,6 +129,9 @@
     "velocity.java": "^1.3.1"
   },
   "resolutions": {
-    "three": "0.118.0"
+    "three": "0.118.0",
```

Comment:
Would this be the new way of dealing with geppetto dependencies in local dev? Do you know if that supports hot-reload if those dependencies changes?